src/GNDStk/plain/src/child.hpp

```diff
@@ -0,0 +1,498 @@
+
+// -----------------------------------------------------------------------------
+// child_t<void,...>
+// For all elements that we see in every existing XML-format GNDS file that
+// we have. The list below was auto-generated by a code we wrote to look for
```

Comment:
This is the kind of thing I think we can auto-generate from the JSON files that specify GNDS. I still need to get those to you.

Reply:
I can't recall if you perhaps gave me those already, or sent a link to where I could find them. My sense is that we'd want to begin with some largely automated process, but probably tweak individual types according to what sorts of functionality users will want for those types. As you know, I designed the entire system so that interfacing with types would be straightforward. And, for any particular chunk of data that someone might want to extract from a GNDS tree, we could have one, or many, different types with which we interface.

Making good, solid types for everything is a big project in and of itself, and I'd suggest leaving that for another pull request, not cluttering this one more.